airbyte-api/src/main/openapi/config.yaml

```diff
@@ -875,16 +875,46 @@ components:
       properties:
         sourceId:
           $ref: "#/components/schemas/SourceId"
+    SourceCreate:
```

Comment:
Shouldn't that be called SourceDefinitionCreate based on https://airbytehq.slack.com/archives/C019WEENQRM/p1601672342057100